tests/ide/recordflux.gpr

```diff
@@ -4,4 +4,8 @@ project RecordFlux is
    for Source_Dirs use (".", "../../specs");
    for Languages use ("Ada", "RecordFlux");
 
+   package Recordflux is
+      for Output_Dir use "../../generated";
+   end Recordflux;
```

Comment:
Could the correct casing for RecordFlux be used here?

Reply:
Seems like GNAT Studio changes the casing when saving `Output_Dir` from the IDE, even though the casing is correct in the package definition in `recordflux.py`. I don't think we can fix that.